tests/api2/test_261_iscsi_cmd.py

```diff
@@ -2377,6 +2397,58 @@ def test_26_resize_target_file(request):
                 assert MB_512 == _read_capacity16(s)
 
 
+@skip_multi_initiator
+def test_27_initiator_group(request):
+    depends(request, ["iscsi_cmd_00"], scope="session")
+
+    initiator_iqn1 = f"iqn.2018-01.org.pyscsi:{socket.gethostname()}:one"
```

Comment:
Minor nit but instead of calling `socket.gethostname()` 3 times, could just call it once since it (shouldn't) be changing between the invocations.